assets/css/_route_ladder.scss

```diff
@@ -22,14 +24,40 @@
 
   .c-route-ladder__dropdown-button {
     height: 2.25rem;
-    width: 2.25rem;
+    width: 1.8125rem;
+    padding: 0;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+    color: black;
+    border-radius: 0.25rem;
+
+    &:active {
+      background-color: unset;
+      color: unset;
+    }
 
     &.dropdown-toggle::after {
-      // Force bootstrap carat to center even with hidden text
-      margin-left: 0;
+      // Hide bootstrap carat
+      display: none;
     }
   }
 
+  .c-route-ladder__dropdown.non-skate-alert .dropdown-menu {
+    width: 20.5rem;
+  }
+
+  .c-route-ladder__dropdown .dropdown-menu {
+    border-radius: 0.25rem;
+    max-width: 20.5rem;
+  }
```

Comment:
Can this section be combined with the `.c-route-ladder__dropdown` section above in a sassy way to look like:

```css
  .c-route-ladder__dropdown {
    display: inline-flex;
    .dropdown,
    .c-route-ladder__alert-icon {
      padding-right: 0.5rem;
    }

    &.non-skate-alert .dropdown-menu {
      width: 20.5rem;
    }

    .dropdown-menu {
      border-radius: 0.25rem;
      max-width: 20.5rem;
    }
  }
```

Reply:
Sure, I've done it in the way you describe, but I've also seen folks on our team avoid it, so what makes one better than the other?